### PR TITLE
correct some InputTags for track collections

### PR DESCRIPTION
**Pull request by lveldere**
- replaced some inputtags for the non-existing collection  "hltIter2HighPtTkMuMerged" with "generalTracks"
- grepped through all Famos menus for "hltIter2HighPtTkMuMerged"  and corrected accordingly
- updated confdb.py accordingly
- tested with runTheMatrix -l 135.1

### CODE REVIEW

**Comment by lveldere** [comment]
please test

**Comment by Martin-Grunewald** [comment]
-1
sorry, i have to do this properly via confb and dump extraction...

**Comment by lveldere** [comment]
@Martin-Grunewald 

As you wish.
Has it any consequence that I don't understand, going via confb and dump extraction?
Or is it just to be absolutely sure that everything is in the right place?

File: HLTrigger/Configuration/python/Tools/confdb.py [hunk 1]
```diff
@@ -456,6 +456,7 @@ def fixForFastSim(self):
       self._fix_parameter(name = 'src',                  type = 'InputTag', value = 'hltHcalTowerNoiseCleaner', replace = 'hltTowerMakerForAll')
 
       # input tags for track collections
+      self._fix_parameter(                               type = 'InputTag', value = 'hltIter2HighPtTkMuMerged', replace = 'generalTracks')
```
Comment (Martin-Grunewald) [inline HLTrigger/Configuration/python/Tools/confdb.py]: This line is already there (few lines down).

**Comment by Martin-Grunewald** [comment]
Please see https://github.com/cms-sw/cmssw/pull/8867

**Comment by lveldere** [comment]
-1

@Martin-Grunewald 
I understood correctly that you take this over in #8867?

**Comment by Martin-Grunewald** [comment]
@lveldere 
Yes.
(It seems I had simply forgotten a commit after a change, as confdb.py is unchanged.)

**Pull request closed without merging**